manifests/params.pp

```diff
@@ -12,5 +12,5 @@
     Sensitive($token.conjur::decrypt)
   }}
   $host_factory_token = undef
-  $version = $conjur_config['version'].lest || { 4 }
+  $version = $conjur_config['version'].lest || { 5 }
```

Comment:
I think this requires doc update. Also, I'm not sure if we should do this lest we bump major version number. This is a potentially breaking change.

Reply:
I suspect we'll just take the the major version bump for this. @jvanderhoof, any thoughts?

I will update the README and changelog accordingly though. Thanks!